pages/linux/upnpc.md

```diff
@@ -0,0 +1,20 @@
+# upnpc
+
+> Configure port forwarding rules on your router via the UPnP protocol.
+> More information: <http://miniupnp.free.fr/>.
```

Comment:
```suggestion
> More information: <https://manned.org/upnpc>.
```
We generally link to the command line documentation or man page, in this case I think the man page would work better here.

Reply:
Hi, thank you for your feedback. The free.fr link was provided by the upnpc tool itself, but I agree with you that the manned.org link is better.